simulation_parameters/Constants.cs

```diff
@@ -25,12 +25,28 @@ public static class Constants
     /// <summary>
     ///   How long the initial compounds should last (in seconds)
     /// </summary>
-    public const float INITIAL_COMPOUND_TIME = 40.0f;
+    public const float INITIAL_COMPOUND_TIME = 30.0f;
 
     public const float MULTICELLULAR_INITIAL_COMPOUND_MULTIPLIER = 1.5f;
 
     public const int FULL_INITIAL_GLUCOSE_SMALL_SIZE_LIMIT = 3;
 
+    /// <summary>
+    ///   Controls how early "Night" starts for initial compound reasons.
+    ///   If this was 0 "Night" would start as soon as light level reaches 0%.
+    ///   This is a fraction percentage of the day.
```

Comment:
```suggestion
    ///   This is a fraction of the day.
```
This is not a percentage. I want to keep percentages and fractions really clear in the code which is which as otherwise there's going to be massive bugs.